### PR TITLE
Use idText for PropertyGetterMethod and PropertySetterMethod

### PR DESCRIPTION
**Pull request by nojaf**
Fixes https://github.com/dotnet/fsharp/issues/15212

### CODE REVIEW

File: src/Compiler/Checking/NameResolution.fs [hunk 1]
```diff
@@ -2078,7 +2078,12 @@ type TcResultsSinkImpl(tcGlobals, ?sourceText: ISourceText) =
 
         let keyOpt =
             match item with
-            | Item.Value vref -> Some (endPos, vref.DisplayName)
+            | Item.Value vref ->
+                if vref.IsPropertyGetterMethod || vref.IsPropertySetterMethod then
+                    // We don't want to skip a symbol if both getter and setter are present.
+                    Some (endPos, vref.Id.idText)
```
Comment (vzarytovskii) [inline src/Compiler/Checking/NameResolution.fs:2084]: Is it possible for `Id` to not be set for ref?

Comment (nojaf) [reply src/Compiler/Checking/NameResolution.fs:2085]: Highly unlikely I think, but I added add a guard for it.

File: tests/service/Symbols.fs [hunk 2]
```diff
@@ -427,3 +427,50 @@ let f2 b1 b2 b3 b4 b5 =
...
             | _ -> ()
+
+module FSharpMemberOrFunctionOrValue =
+    [<Test>]
+    let ``Both Set and Get symbols are present`` () =
+        let _, checkResults = getParseAndCheckResults """
+namespace Foo
+
+type Foo =
+    member _.X
```
[the remark this answers is not in the document]
Comment (auduchinok) [reply tests/service/Symbols.fs:438]: Cool, thanks!

File: tests/service/ProjectAnalysisTests.fs [hunk 3]
```diff
@@ -3443,27 +3443,33 @@ let ``Test Project24 all symbols`` () =
             ("``.ctor``", "file1", ((4, 5), (4, 23)), ["defn"], ["member"; "ctor"]);
             ("NameGetSet", "file1", ((5, 13), (5, 23)), ["defn"], ["member"; "getter"]);
             ("int", "file1", ((7, 20), (7, 23)), ["type"], ["abbrev"]);
+            ("NameGetSet", "file1", ((5, 13), (5, 23)), ["defn"], ["member"; "setter"]);
             ("NameGet", "file1", ((9, 13), (9, 20)), ["defn"], ["member"; "getter"]);
             ("int", "file1", ((11, 20), (11, 23)), ["type"], ["abbrev"]);
+            ("NameGet", "file1", ((9, 13), (9, 20)), ["defn"], ["member"; "setter"]);
```
[the remark this answers is not in the document]
Comment (nojaf) [reply tests/service/ProjectAnalysisTests.fs:3449]: Yes, the name is rather misleading. `NameGet` actually has both a getter and a setter.

**Review by edgarfgp — APPROVED** [review]

**Pull request merged**